panel/src/main/scala/hmda/panel/PanelCsvParser.scala

```diff
@@ -71,4 +70,15 @@ object PanelCsvParser extends InstitutionComponent with WriteInstitutionProtocol
 
   private def byte2StringFlow: Flow[ByteString, String, NotUsed] =
     Flow[ByteString].map(bs => bs.utf8String)
+
+  private def createInstitutionDB() {
+    println("Creating new schema...")
```

Comment:
I know this is a console app, but could we add proper logging here as opposed to `println`?